orchagent/dash/dashvnetorch.cpp

```diff
@@ -446,11 +448,6 @@ bool DashVnetOrch::addVnetMap(const string& key, VnetMapBulkContext& ctxt)
         }
 
         remove_from_consumer = addOutboundCaToPa(key, ctxt);
-        // If addOutboundCaToPa fails, skip addPaValidation
```

Comment:
Why is this removed?

Reply:
There was a logic error in the existing code, moving `addPaValidation` to inside `addOutboundCaToPa` should fix it. @prabhataravind for viz

Reply:
Could you add a comment in addVnetMap that addPaValidation is being called in addOutboundCaToPa to avoid dependency issues?

Reply:
Added the comment in addVnetMap